### ISSUE TITLE
Accessibility: MAS40B: Inspect: 'Name' property is empty for "Debugging ToolBar" Panel.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/vscode#17170]
VS Code Version:1.6.1
OS Version:1607

**Steps to Reproduce:**
1.Open Visual studio code editor.
2.Using "Ctrl+Shift+D" key navigate to the debug side view bar option.
3.Open the launch.json file under the "Configure or Fix: launch.json"button and select "Node.js" from the drop down list.
4. Add configurations to "launch.json" utilizing IntelliSense with CTRL+SPACE.
5. Start debugging by pressing F5.
6. Using Inspect, check 'Name' property for "Debugging Tool Bar" Panel.

**Expected Results:**
"Name" property should not be empty. Pane should have a meaningful and appropriate name.
Eg: Name property should be "Debugger Options Toolbar Panel".

**Actual Result:**
' 'Name' property is empty for the "Debugging ToolBar Panel.
   Note: 'IsKeyboardFocusable' property is also FALSE for the same "Debugging ToolBar" Panel.
**User Impact:**
Users relying on screen readers will get confused if the name property is not meaningful and inappropriate.

 
![debugging toolbar name and focus](https://cloud.githubusercontent.com/assets/22956130/21183252/2143254e-c22d-11e6-96aa-d5cf7193584a.png)

**Comment by isidorn** [comment]
This element can not be focused by designed. As user focus into the debug toolbar the focus will automatically jump to the first item. I believe this is good behavior as we save one tab press by not focusing the parent container.

Let me know if focusing the parent is optimal in this case and I can reopen. IMHO we are doing the right thing now -> closing
Currently we have the same behavior as the user tabs from viewlet to the top level actions there - we do not first focus the "viewlet toolbar"

fyi @bpasero

**Comment by WiproPSAccTest** [comment]
As described above  the debug tool bar will not get keyboard focus. User can access those options using well defined shortcut keys. It can be treated as by design.
